Code/src/main/java/nl/utwente/viskell/ui/CustomUIPane.java

```diff
@@ -61,6 +62,12 @@ public CustomUIPane(HaskellCatalog catalog) {
         this.addEventHandler(KeyEvent.KEY_PRESSED, this::handleKey);
     }
 
+    /** @return modifiable list of children. */
+    @Override 
+    public ObservableList<Node> getChildren() {
+        return super.getChildren();
+    }
+    
```

Comment:
Because it is protected by default, and we directly manipulate UIPane parts (which is a bit ugly).